pyproject.toml

```diff
@@ -305,6 +287,12 @@ dependencies = [
 [tool.uv.sources]
 marimo_docs = { path = "./docs", editable = true }
 
+[tool.uv.build-backend]
+module-name = "marimo"
+module-root = ""
+source-include = ["third_party.txt", "third_party_licenses.txt"]
+wheel-exclude = ["marimo/_smoke_tests/**"]
```

Comment:
1. do we need to includ `"marimo/_static/", "marimo/_lsp/"`?
2. we will need to update `scripts/modify_pyproject_for_marimo_base.sh` to work with this new format. this script strips the assets (except index.html) to reduce the size of the build

Reply:
> do we need to includ "marimo/_static/", "marimo/_lsp/"?

these are automatically included

```sh
unzip -l dist/marimo-0.14.13-py3-none-any.whl | grep static | wc -l
     294
```

Reply:
> we will need to update scripts/modify_pyproject_for_marimo_base.sh to work with this new format. this script strips the assets (except index.html) to reduce the size of the build

I just pushed a new script for this and updated the associated workflows